flixel/graphics/frames/bmfont/BMFontInfo.hx

```diff
@@ -107,17 +107,19 @@ class BMFontInfo
 	static function fromBytes(bytes:BytesInput)
 	{
 		final blockSize = bytes.readInt32();
+		final size = bytes.readInt16();
 		final bitField = bytes.readByte();
+		final charsetByte = bytes.readByte();
 		final fontInfo:BMFontInfo =
 		{
-			size: bytes.readInt16(),
+			size: size,
 			smooth: (bitField & 0x80) != 0,
 			unicode: (bitField & (0x80 >> 1)) != 0,
 			italic: (bitField & (0x80 >> 2)) != 0,
 			bold: (bitField & (0x80 >> 3)) != 0,
 			fixedHeight: (bitField & (0x80 >> 4)) != 0,
-			charset: String.fromCharCode(bytes.readByte()),
-			stretchH: bytes.readInt16(),
+			charset: charsetByte > 0 ? String.fromCharCode(charsetByte) : "",
```

Comment:
Prevent getting a charset that is a single character string containing the \0 null character

Reply:
Just out of curiosity, do you know what the charset byte is for? I can't seem to find much info about it online

Reply:
It's not super clear exactly what this is for. The text portion of the spec shows:

`charset | The name of the OEM charset used (when not unicode).`

But they don't give a list of values, or where you can get more information on that. You can find pages like [this](https://en.wikipedia.org/wiki/Code_page) that describe "OEM code pages", but their ID's aren't small enough to fit in a single byte... so it's unclear exactly what they mean from the searching I've done.